doc/README.md

```diff
@@ -19,52 +19,50 @@ to learn more about the format that is used.
 The documentation for the standard library is generated from
```

Comment:
The closing parenthesis for the comment "see INSTALL to learn more..." should be after "... the format that is used."  Or it could a sentence on its own without parentheses.